CHANGELOG.unreleased.md

```diff
@@ -19,7 +19,9 @@ Section headings should be at level 3 (e.g. `### Added`).
 - Added `wandb.public.Project.id` property to support fetching the project ID on a `wandb.public.Project` (@tonyyli-wandb in https://github.com/wandb/wandb/pull/9194).
 - Added new `wandb.sdk.automations` subpackage to support programmatic management of W&B Automations (@tonyyli-wandb in https://github.com/wandb/wandb/pull/8935).
```

Comment:
is this actually something that should be in the changelog? is it an information that you want to communicate to the user or is it an internal logic that could move in the future